aten/src/ATen/native/mkldnn/xpu/Blas.cpp

```diff
@@ -356,9 +292,8 @@ Tensor& bmm_out(const Tensor& self, const Tensor& batch2, Tensor& result) {
     return result;
   }
 
-  if (self.is_complex() || self.scalar_type() == ScalarType::Double) {
-    TORCH_CHECK(
-        false, "Double and complex datatype matmul is not supported in oneDNN");
+  if (self.is_complex()) {
+    AT_ERROR("Complex datatype matmul is not supported in oneDNN");
```

Comment:
`AT_ERROR` has been deprecated.

Reply:
Thanks for suggestion, `AT_ERROR` in file is changed to `TORCH_CHECK`